payjoin/src/core/version.rs

```diff
@@ -28,6 +28,20 @@ pub enum Version {
     Two = 2,
 }
 
+impl Version {
+    /// Returns the string representation of the version
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            Version::One => "1",
+            Version::Two => "2",
+        }
+    }
+}
+
+impl From<Version> for &'static str {
+    fn from(val: Version) -> Self { val.as_str() }
+}
+
```

Comment:
I don't love this addition because it seems like it's just needless complexity for a tiny aesthetic change above